src/chatkit.ts

```diff
@@ -147,6 +147,16 @@ export interface User {
   customData?: any;
 }
 
+export interface SendMessageOptions {
+  sender_id: string;
```

Comment:
Could we make this `senderId` please? Camel case is what we are settling on for our JS-based SDKs (I know there are a few exceptions in this repo!)

Reply:
ok, sure :)